libr/flags/flags.c

```diff
@@ -251,6 +251,25 @@ static RFlagItem *evalFlag(RFlag *f, RFlagItem *item) {
 	return item;
 }
 
+/* return true if flag.* exist at offset. Otherwise, false is returned.
+ * For example (f, "sym", 3, 0x1000)*/
+R_API bool r_flag_exist_at(RFlag *f, const char *flag_prefix, ut16 fp_size, ut64 off) {
```

Comment:
No. As I understood r_flag_get_at "returns the last flag item defined before or at the given offset". I don't need to know last flag at the given offset or before, I want to know if the given flag exist at the given offset. From my point of view it is 2 completely different functions. Maybe I just don't understand how to use r_flag_get_at?